docusaurus/docs/how-to-guides/data-source-plugins/add-support-for-externally-shared-dashboards.md

```diff
@@ -0,0 +1,61 @@
+---
+id: add-support-for-externally-shared-dashboards
+title: Add support for externally shared dashboards
+description: How to add support for externally share dashboards (previously called Public dashboards).
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - externally shared dashboards
+  - public dashboards
+  - data source
+  - datasource
+---
+
+Externally shared dashboards (previously called Public dashboards) retrieve the query from the backend, instead of receiving it from the frontend. This is to avoid exposing sensitive data and performing unauthorized queries.
```

Comment:
```suggestion
[Externally shared dashboards](https://grafana.com/docs/grafana/latest/dashboards/share-dashboards-panels/shared-dashboards/#externally-shared-dashboards) (previously called Public dashboards) allow Grafana users to share access to their dashboards with anyone, without needing to add them to their Grafana organization as a user. When a dashboard is accessed in this way, it retrieves the query from the backend, instead of receiving it from the frontend. This is to avoid exposing sensitive data and performing unauthorized queries.
```

It might help to provide a link back to the main docs and briefly describe the feature.